template/src/Station.tsx

```diff
@@ -35,6 +36,14 @@ export default function Station() {
             <b>{address}</b>
           </p>
           <p>Balance: {balance} uluna</p>
+
+          {
+            <SendToken
```

Comment:
after connection to an wallet a list of buttons with all supported tokens will appear, I do not check if the balance is also available.

Errors should been shown in the gui instead of the console.